opcua/server/address_space.py

```diff
@@ -108,12 +108,13 @@ def _is_suitable_ref(self, desc, ref):
     def _suitable_reftype(self, ref1, ref2, subtypes):
         """
         """
+        if not subtypes and ref2.Identifier == ua.ObjectIds.HasSubtype:
+            return False
         if ref1.Identifier == ref2.Identifier:
             return True
-        #TODO: Please check the changes here
```

Comment:
References are node I'd, this test is a bit wrong we should compare entire
node ID, not only identifier.. Probably a performance optimization, but not
sure we should keep it

On Fri, Jul 15, 2016, 08:48 Denis Štogl <notifications@github.com> wrote:

> In opcua/server/address_space.py
> <https://github.com/FreeOpcUa/python-opcua/pull/244#discussion_r70928720>:
>
> >          if ref1.Identifier == ref2.Identifier:
> >              return True
> > -        #TODO: Please check the changes here
>
> Looks good. But which type is the reference now
>
> —
> You are receiving this because you authored the thread.
> Reply to this email directly, view it on GitHub
> <https://github.com/FreeOpcUa/python-opcua/pull/244/files/c2997ee05024355fd8dc3ac027f7654916199063#r70928720>,
> or mute the thread
> <https://github.com/notifications/unsubscribe-auth/ACcfzisZcWXQIFSSVx7VBSotkgmdjj_Yks5qVy1KgaJpZM4JLHjh>
> .
>